### DISCUSSION TITLE
file upload with firefox does not work, with edge it is ok

### DISCUSSION THREAD

**Discussion by Anirah1** [community/community#23322] [New to GitHub]
<p>file upload with firefox does not work, with edge it is ok. What may be the reason, with firefox?</p>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi there,</p>
<p>There are few reasons one could be experiencing this sort of trouble.</p>
<ol>
<li>
<p>This can sometimes be caused by conflicting browser extensions. Could you please try visiting the page again with browser extensions disabled or in an incognito/private browsing window?</p>
</li>
<li>
<p>In order for <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> to function properly it requires JavaScript to be enabled. Could you please confirm that JavaScript is enabled in your browser?</p>
</li>
<li>
<p>This can sometimes occur when accessing GitHub using an outdated browser.   We would recommend veryfying that you have the latest release of Firefox.</p>
</li>
</ol>
<p>With any luck, one of these options should resolve this issue, but please let us know if you continue to run into any further trouble.</p>
<p>If you could confirm the exact browser version and operating system you’re using as well as share any relevant screenshots with non-identifiable information, that would be most appreciated.</p>
<p>It would also be very helpful for us to see a screenshot of the Web Console view in your web browser. For example, using Firefox:</p>
<p><a href="https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console" rel="nofollow noopener">https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console</a></p>
<p>Hopefully there will be something in either the console or network logging that can help shed some light on this.</p>
<p>Once we have more information, we’ll certainly do our best to help!</p>
